components/StayingPopulation.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <v-col cols="12" md="6">
+    <div class="StayingPopulation">
+      <div class="StayingPopulation-title">
+        {{ $t('●緊急事態宣言中の\n都内の滞在人口の増減状況') }}<br />
```

Comment:
```suggestion
        {{ $t('●緊急事態宣言中の都内の滞在人口の増減状況') }}<br />
```

`\n` はとりましょうか。微妙に隙間ができてしまっています。

<img width="581" alt="スクリーンショット 2021-01-29 16 13 57" src="https://user-images.githubusercontent.com/14883063/106243803-4be16a80-624d-11eb-970c-6947042e9c4d.png">

Reply:
対応しました